docs/contributions-flow.md

```diff
@@ -37,17 +69,31 @@ subscriptions.setOnLoginRequest(function() {
 });
 ```
 
-If a user elects for a presented offer, SwG will run the [Subscribe flow](./subscribe-flow.md).
-
-
-## Contribution options
-
-The contributions API (`showContributions`) accepts a list of SKUs to be displayed.
-
-For instance:
-
+## Contribution response
+The response returned by the `setOnContributionResponse` callback is the [`SubscribeResponse`](../src/api/subscribe-response.js) object. It includes purchase details, as well as user data.
+### Structure
+The SubscriptionResponse object has the following structure:
 ```
-subscriptions.showContributions({skus: ['sku1', 'sku2']});
+{
+  "raw": "",
+  "purchaseData" : {
+    "raw": "",
+    "signature": "",
+  },
+  "productType": "UI_CONTRIBUTION",
```

Comment:
Update subscriptions flow to include product type in doc.

Reply:
Done.